modules/exploits/multi/http/mantisbt_php_exec.rb

```diff
@@ -190,13 +195,16 @@ def upload_xml(payload_b64, rand_text, cookies, is_check)
     data_post = data.to_s
 
     print_status('Sending payload...')
-    return send_request_cgi({
+    res = send_request_cgi({
```

Comment:
Missed that one :) please wait with merging this I would like to take a look at the issue from the referenced issue